docs/HowTo/Reconstruct-Historical-States-Service.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Reconstruct Historical States Service
+---
+
+# Reconstruct Historical States Service
+
+!!! warning
+
+    Reconstruct historical states service is an early access feature and is subject to change.
+
+An archive node is able to reconstruct historical states from genesis up to the current checkpoint. This service runs during start up.
```

Comment:
I think the key detail we want to explain here is that this allows you to create a full archive node while also using checkpoint sync.  Also worth mentioning that generating states this way is faster as many validations can be safely skipped and because it avoids weak subjectivity concerns (https://docs.teku.consensys.net/en/latest/Concepts/Weak-Subjectivity/).

It's probably worth mentioning this feature on that weak subjectivity page as well.